tests/test_endpoints.py

```diff
@@ -103,3 +103,25 @@ def test_nhs_login_p9(nhsd_apim_proxy_url, nhsd_apim_auth_headers):
         headers=headers
     )
     assert resp.status_code == 200
+
+
+@pytest.mark.smoketest
+@pytest.mark.auth
+@pytest.mark.integration
+@pytest.mark.user_restricted_separate_nhs_login
+@pytest.mark.nhsd_apim_authorization({"access": "patient", "level": "P9", "login_form": {"username": "9912003071"}})
+def test_prism_returns_external_file(nhsd_apim_proxy_url, nhsd_apim_auth_headers):
+    headers = {
+        "Interaction-ID": "urn:nhs:names:services:gpconnect:fhir:operation:gpc.getstructuredrecord-1",
+        "accept": "*/*",
+        "X-Correlation-ID": "11C46F5F-CDEF-4865-94B2-0EE0EDCC26DA",
+        "X-Request-ID": "60E0B220-8136-4CA5-AE46-1D97EF59D068"
+    }
+    headers.update(nhsd_apim_auth_headers)
+
+    resp = requests.get(
+        f"{nhsd_apim_proxy_url}/Patient/9000000009/MedicationRequest",
+        headers=headers
+    )
+    expected_response = load_example_json("GetResponsePreviousPrescriptionDetailed.yaml")
```

Comment:
You are passing a yaml file into `load_example_json` function. Something is wrong :)